packages/headless/src/features/configuration/configuration-slice.test.ts

```diff
@@ -81,6 +82,20 @@ describe('configuration slice', () => {
       const action = updateBasicConfiguration({platformUrl});
       expect('error' in action).toBe(false);
     });
```

Comment:
Would it be useful to have a test for when a search url is a relative url, ensuring that the regex is not transforming it when determining the analytics url? Or is there a test already?